stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -188,7 +188,7 @@ public class MessageListViewModel @JvmOverloads constructor(
                 domain.useCases.sendMessage(event.message).enqueue()
             }
             is Event.MessageReaction -> {
-                onMessageReaction(event.message, event.reactionType)
+                onMessageReaction(event.message, event.reactionType, event.enforceUnique)
```

Comment:
Took me a solid few minutes to figure out what kind of `event` we're talking about here - we should _really_ rename these to `Action`s at some point 😅

Reply:
Agree, I'll add a task for that

Reply:
https://stream-io.atlassian.net/browse/CAS-572